docs/extras/fastapi.md

```diff
@@ -11,3 +11,15 @@ The FastAPI extra node can be used by adding the following to `pyproject.toml`
 ## API Reference
 
 ::: aineko.extras.fastapi.main.FastAPI
+
+# Health Check Endpoint
+
+Aineko comes with a light weight helper function as follows:
+
+::: aineko.extras.fastapi.health.read_health
+
+## Authentication Considerations
+
+By default, the Aineko health endpoint does not contain authentication. 
```

Comment:
```suggestion
By default, the `/health` endpoint does not contain authentication. 
```
or just
```suggestion
By default, the `health` endpoint does not contain authentication. 
```

Imo we should drop the Aineko here.

Reply:
updated here: [42dc84c](https://github.com/aineko-dev/aineko/pull/142/commits/42dc84c8995f32e0b2a788e40c2903ee5a29eb10)